code/gui/mainwindow.cpp

```diff
@@ -20,29 +20,69 @@ MainWindow::MainWindow(QWidget *parent, const char *title) :
 
 
     // Setup slot connections
-    connect(ui->setup_actuator, SIGNAL(triggered()), this, SLOT(OpenActuatorSetup()));
+    connect(ui->setup_actuator, SIGNAL(triggered()), this, SLOT(openActuatorSetup()));
+
+	// setup focus and an event filter to capture key events
+	this->installEventFilter(this);
+	this->setFocus();
 }
 
-QTextEdit* MainWindow::GetLogView()
+bool MainWindow::eventFilter(QObject *object, QEvent *event)
 {

```

Comment:
Don't remember if we have anything in the guidelines about inline brackets, but I think there was something like that.